exporter/kafkaexporter/README.md

```diff
@@ -61,6 +61,13 @@ The following settings can be optionally configured:
   User should calculate this as `num_seconds * requests_per_second` where:
     - `num_seconds` is the number of seconds to buffer in case of a backend outage
     - `requests_per_second` is the average number of requests per seconds.
+- `compression` (default = none): The compression codec to compress the messages sent to Kafka. Four compression methods are supported: `gzip`, `snappy`, `lz4` and `zstd`.
+- `compression_level` (default = -1000): The corresponding compression level which measures the compression degree.
```

Comment:
A default of `-1000` does not make sense when the valid range for, say, `gzip`, is 1 to 9. Maybe you need to specify a default for each compression method?

Reply:
Actually, the default value is specified by sarama. -1000 means we don't use any compression level. So I think it may be better to follow their spec?

Reply:
Ah, okay, so there is no default for each compression method, my bad 👍